src/bin/pg_autoctl/service_keeper.c

```diff
@@ -802,6 +807,12 @@ keeper_node_active(Keeper *keeper, bool doInit)
 		}
 	}
 
+	/*
+	 * Finally make establish a connection for notifications in case it had
+	 * closed before
+	 */
+	(void) pgsql_listen(&(keeper->monitor.notificationClient), &((char *){ 0 }));
```

Comment:
Same as before, I'm not happy with this notation, let's make it a whole lot more explicit please?